XpringKit/Org_Xrpl_Rpc_V1_Transaction.PathElement+XRPPathElement.swift

```diff
@@ -0,0 +1,7 @@
+internal extension XRPPathElement {
+  init(pathElement: Org_Xrpl_Rpc_V1_Payment.PathElement) {
+    self.account = pathElement.hasAccount ? pathElement.account.address : nil
+    self.currency = pathElement.hasCurrency ? XRPCurrency(currency: pathElement.currency) : nil
+    self.issuer = pathElement.hasIssuer ? pathElement.issuer.address : nil
+  }
+}
```

Comment:
is it ok for these fields of XRPPathElement to be optional?  I see that in the proto none of account, currency or issuer are optional?

Reply:
There are no required fields in proto3:
https://github.com/apple/swift-protobuf/blob/master/Documentation/API.md#type-modifiers

Instead there are default values:
https://github.com/apple/swift-protobuf/blob/master/Documentation/API.md#type-modifiers

As such, we check the `hasFoo` to decide if these should be set or not. 
-----

A good way to think about this is destinationTags. 
- If you didn't set a destination tag, then `.destinationTag` returns 0 (default value)
- If you set a destination tag to 0, then `.destinationTag` returns 0 (the set value)

These are useful to distinguish between.